pyodide-build/pyodide_build/build_env.py

```diff
@@ -285,10 +293,12 @@ def pyodide_tags() -> Iterator[Tag]:
     """
     PYMAJOR = get_pyversion_major()
     PYMINOR = get_pyversion_minor()
-    PLATFORM = platform()
+    # Accept both "pyodide_abiver_wasm32" and "emscripten_ver_wasm32" (rust packages)
+    platforms = [platform(), emscripten_platform()]
```

Comment:
This should probably be documented somewhere on the page for loading packages. Maybe we could do a separate subsection on wheel compatibility?